samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -410,14 +443,15 @@ static int presence_detected_get(struct bt_mesh_sensor_srv *srv,
 				 struct bt_mesh_sensor_value *rsp)
 {
 	int err = bt_mesh_sensor_value_from_micro(sensor->type->channels[0].format,
-						  pres_detect * 1000000LL, rsp);
+						  BASE_UNITS_TO_MICRO(pres_detect ? 1 : 0), rsp);
 
-	if (err) {
+	if (err && err != -ERANGE) {
 		printk("Error encoding presence detected (%d)\n", err);
+		return err;
 	} else {
```

Comment:
`else` is no longer needed with early return

Reply:
seems compliance skipped this one! fixed now.